src/glyph/serialize.rs

```diff
@@ -1,11 +1,11 @@
 //! Writing out .glif files
 
-use std::io::{Cursor, Write};
-
 use quick_xml::{
     events::{BytesDecl, BytesEnd, BytesStart, BytesText, Event},
     Error as XmlError, Writer,
 };
+use std::f32;
```

Comment:
hmm, did cargo fmt do this? The 'standard' ordering of imports is std -> extern crates -> internal types; although this is just a convention.

Reply:
`cargo fmt` does not change this, so uh?